---
title: "Jacob Arie's Resume"
author: Jacob Arie van Diermen
date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
# uncomment this line to produce HTML and PDF in RStudio:
knit: pagedown::chrome_print
---

```{css, echo=FALSE}
/* eliminates the sidebar after page 1 */
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.2in;
}

.details, .date {
    break-inside: avoid-column;
}

.allowstrong strong,
.allowstrong b {
  font-weight: 700;
}
```

# Aside

![Jacob Arie](assets/jacob_arie_cv.png){width="80%"}

## Contact Info {#contact}

- <i class="fa fa-envelope"></i> [jarivandiermen\@hotmail.com](mailto:jarivandiermen@hotmail.com){.email}
- <i class="fa fa-github"></i> [Github](https://github.com/Jari-van-Diermen)
- <i class="fa-brands fa-docker"></i> [Docker Hub](https://hub.docker.com/u/jarivdiermen)
- <i class="fa-brands fa-linkedin"></i> [Linked in](https://www.linkedin.com/in/jari-van-diermen-99a49712a/)
- <i class="fa fa-phone"></i> +31 653 270 827
- For more information, please contact me via email.

## Skills {#skills}

#### Languages

- <i class="fa-solid fa-circle"></i> Fluent Dutch (Native language)
- <i class="fa-solid fa-circle"></i> Fluent English (C2 level)

#### Programming languages

- <i class="fa-solid fa-circle"></i> R
- <i class="fa-solid fa-circle"></i> Python
- <i class="fa-solid fa-circle"></i> Bash
- <i class="fa-solid fa-circle"></i> SLURM
- <i class="fa-solid fa-circle"></i> LaTeX
- <i class="fa-solid fa-circle"></i> Nextflow
- <i class="fa-solid fa-circle"></i> Snakemake

# Main

## Jacob Arie van Diermen {#title}

### MSc

## Experience {data-icon="laptop"}

### PhD candidate

Hubrecht institute, developmental epigenomics (Jop Kind Group)

Utrecht, The Netherlands

2026/03 - 2024/09

<br>

As a PhD candidate, I conducted research to better understand the fundamental
principles of genome regulation during development. This taught me how to process,
analyze and present large amounts of genomics data in HPC environments. Moreover, I developed pipelines
for preprocessing multimodal genomics data ([sciT-snakemake pipeline](https://github.com/Jari-van-Diermen/SciT_snakemake))
and an Accompanying R-package ([LoadSciLooms](https://github.com/Jari-van-Diermen/LoadSciLooms)),
both of which aid the lab in its research efforts.

### Research assistant

Karolinska Institutet, Clinical Neurosciences (Robert Harris Group)

Solna, Sweden

2024/03 - 2023/07

<br>

::: {.allowstrong}
As a continuation of my work as a research intern, I improved the computational
pipeline for evolutionary inference with improved parallelization, data cleaning, documentation
and **Docker containerization** for reproducibility. Additionally, I developed a
[data dashboard using **R shiny**](https://jari-van-diermen-shiny-app-acomys.share.connect.posit.cloud/)
to improve exploration of the pipeline results. This allowed me to further improve my **Python**, **R**
and **Bash** programming expertise 
:::

### Teaching assistant

Utrecht university, Department of Biology

Utrecht, The Netherlands

2023/05 - 2023/02

<br>

As a teaching assistant for the course `Kwantitatieve biologie (B-B1KWBI20)`,
I helped first-year biology students grasp statistical concepts and R programming.

### Research intern

Karolinska Institutet, Clinical Neurosciences (Robert Harris Group)

Solna, Sweden

2023/03 - 2022/05

<br>

::: {.allowstrong}
As a research intern, I have created a computational pipeline that uses evolutionary
inference to better predict which genes are relevant for the regenerative
wound-healing in the African Spiny mice. I gained extensive experience in
**high performance computing (SLURM)**, programming (**R**, **Python** and **Bash**) and
version control (**Git**, **Github**). Additionally, I learned to present and report my findings.
:::

### Research intern

Utrecht university, Cell biology, neurobiology and biophysics,
Anna Akhmanova Group (Cellular dynamics)

Utrecht, The Netherlands

2022/03 - 2021/01

<br>

As a research intern, I studied proteins affecting cellular structure. This
taught me extensive lab skills, in addition to planning and presenting
experimental results.

## Education {data-icon="graduation-cap" data-concise="true"}

### Utrecht university, Graduate School of Life Sciences

MSc Molecular and Cellular life sciences (MCLS), specialization Biophysics &
Molecular Imaging, Bioinformatics minor profile

Utrecht, The Netherlands

2023 - 2020

Bioinformatics minor profile

- *Thesis: Comparative genomics approach finds DNA-repair and immune-related genes
in the regenerative African spiny mouse*

Major research project

- *Thesis: Characterization of CSPP1 interaction with microtubules in cells*

### Utrecht university, Department of Biology

BSc in Biology

Utrecht, The Netherlands

2020 - 2017

**graduated with honors (cum laude)**

## <i class="fa-solid fa-box-archive"></i> Additional skills {data-icon=NULL}

### Software skills

N/A

N/A

::: {.allowstrong}
- **Containerization:** Docker, Singularity
- **Workflow management:** Snakemake, Nextflow
- **Version control:** Git, Github
- **Programming:** R, Rstudio, Python, Visual studio code, Bash, Zsh, LaTeX
- **Package management:** Conda (miniconda, miniforge)
- **OS:** Linux (i.e. Debian, Ubuntu)
- **High performance computing (HPC):** SLURM
- **Evolutionary inference:** HyPhy (aBSREL, MEME), PAML codeml
- **Image analysis:** Fiji/ ImageJ
:::
